temporal-serviceclient/src/main/java/io/temporal/serviceclient/WorkflowServiceStubsOptions.java

```diff
@@ -174,6 +181,11 @@ public Duration getRpcQueryTimeout() {
     return rpcQueryTimeout;
   }
 
+  /** @return frequency at which connection backoff should be reset. */
```

Comment:
Add that it can return null if reset is off

Reply:
Done